integration-tests/common/common.go

```diff
@@ -115,78 +124,60 @@ func stripKeyPrefix(key string) string {
 	return key
 }
 
-func New(env string, isK8s bool) *Common {
-	var err error
+func New(testConfig *tc.TestConfig) *Common {
 	var c *Common
-	if env == "devnet" {
-		c = &Common{
-			IsK8s:     isK8s,
-			ChainName: ChainName,
-			ChainId:   DevnetChainID,
-			SolanaUrl: SolanaDevnetURL,
-		}
-	} else {
-		c = &Common{
-			IsK8s:     isK8s,
-			ChainName: ChainName,
-			ChainId:   LocalnetChainID,
-			SolanaUrl: SolanaLocalNetURL,
-		}
+
+	duration, err := time.ParseDuration(*testConfig.OCR2.TestDuration)
```

Comment:
is this needed? I think this field is defined as either `time.Duration` or `blockchain.StrDuration` which will return error if it cannot unmarshall it. Can you check?